native-schema-registry/shared/protos/◉◉◉unicode⏩.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
```

Comment:
Are all these .proto files only for tests? If so, should they go into a test folder?

Reply:
these are only used for tests for now, but it will be used between all languages so Ravi suggested me to put them here.

Reply:
Good point. Please add a sub-folder test to indicate that these are for tests.

Reply:
refactored